src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,58 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private K[] keys = (K[]) new Object[this.size];
+    private V[] values = (V[]) new Object[this.size];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is default for int